src/main/resources/templates/db_scripts/updateItemStatusDate.sql

```diff
@@ -0,0 +1,23 @@
+
+-- Updates item status date if status name was changed.
+-- Item status date stores with timezone.
+CREATE OR REPLACE FUNCTION ${myuniversity}_${mymodule}.update_status_date() RETURNS TRIGGER
+AS $$
+  DECLARE
+	  newStatus text;
+  BEGIN
+  	newStatus = NEW.jsonb->'status'->>'name';
+  IF newStatus IS NULL THEN  RETURN NEW;
+  END IF;
+	IF (TG_OP='UPDATE') THEN
```

Comment:
Do we need this check, since the trigger is already before update?

Reply:
Yes, you are right. Thanks!